components/popover/src/Popover.js

```diff
@@ -33,7 +34,10 @@ export const Popover = ({
 	} = useBrand();
 
 	const [open, setOpen] = useState(isOpen);
-	const [position, setPosition] = useState({ placement: 'top', empty: true });
+	const [position, setPosition] = useState({
+		placement: placement ? placement : 'top',
+		empty: !!!placement,
```

Comment:
Isn't this just the same as a single `!` ?

Reply:
yeah but I like to make sure :D